app/backplane/radio_module/src/c_lora_to_udp_tenant.cpp

```diff
@@ -0,0 +1,22 @@
+#include "c_lora_to_udp_tenant.h"
+#include "c_radio_module.h"
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(CLoraReceiveTenant);
+
+void CLoraToUdpTenant::Startup() {
+}
+
+void CLoraToUdpTenant::PostStartup() {
+}
+
+void CLoraToUdpTenant::Run() {
```

Comment:
Won't this screw you up if this is one tenant of a larger group for a task? CTask will call Run() which will get stuck spinning here and the for loop of CTask::Run won't make it to the next tenant?

Reply:
not a problem if it ends up in a task all to its own but probably worth at least writing a comment saying dont use this in part of a larger group

Reply:
Actually was not the intention. Good catch!